sdk/keyvault/Azure.Security.KeyVault.Administration/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Release History
 
-## 4.0.0-beta.6 (Unreleased)
+## 4.0.0 (2021-06-15)
 
-### Breaking Changes
+- Initial release of Azure.Security.KeyVault.Administration
+- Added `KeyVaultAccessControlClient` to managed role assignments and definitions for Managed HSM.
+- Added `KeyVaultBackupClient` to backup and restore Managed HSM.
```

Comment:
These seem like sub-bullets of the first one

Reply:
I felt they were more specific, but could change the first bullet to something like "... including KVACC and KVBC."